CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Pending
 
+- [\#567](https://github.com/arkworks-rs/algebra/pull/567) (`ark-ec`) Allow to overwrite the default implementation of the `msm` function provided by the `VariableBaseMSM` trait by a specialized version in `SWCurveConfig`.
```

Comment:
Let's move this to the 0.4 section (as that's still unreleased fully)

Reply:
Sure! I moved it to improvements there, also since it is not a breaking change?